pyxem/utils/expt_utils.py

```diff
@@ -130,6 +117,8 @@ def radial_average(z, center,cython=True):
     radial_profile : array
         Radial profile of the diffraction pattern.
     """
+    center = (z.shape[0],z.shape[1]) #geometric shape work
```

Comment:
Also here a factor of 2?

Reply:
Yes again, but this may well be rewritten